src/containers/find-offer/offer-card-square/OfferCardSquare.tsx

```diff
@@ -103,20 +105,22 @@ const OfferCardSquare: FC<OfferCardSquareProps> = ({
         </AppChip>
       </Box>
 
-      <Box sx={styles.priceContainer}>
-        <TitleWithDescripiton
-          description={t('common.hourSlash')}
-          style={styles.titleWithDescription}
-          title={`${price} ${t('common.uah')}`}
-        />
-        <Box>
-          <AppRatingMobile
-            reviewsCount={author.totalReviews}
-            value={authorAvgRating}
+      <Box>
```

Comment:
Why do you wrap these components additional Box? Delete it. This is redundant

Reply:
this is for better card appearance and its responsibility, this block with price and buttons should always be at the bottom regardless of the card size.

Reply:
I think we should handle it without an additional block.